servers/Azure.Mcp.Server/docs/azmcp-commands.md

```diff
@@ -1905,13 +1905,13 @@ azmcp virtualdesktop hostpool list --subscription <subscription> \
 
 # List session hosts in a host pool
 # ❌ Destructive | ✅ Idempotent | ❌ OpenWorld | ✅ ReadOnly | ❌ Secret | ❌ LocalRequired
-azmcp virtualdesktop hostpool sessionhost list --subscription <subscription> \
+azmcp virtualdesktop hostpool session list --subscription <subscription> \
                                                [--hostpool <hostpool-name> | --hostpool-resource-id <hostpool-resource-id>] \
                                                [--resource-group <resource-group>]
 
 # List user sessions on a session host
 # ❌ Destructive | ✅ Idempotent | ❌ OpenWorld | ✅ ReadOnly | ❌ Secret | ❌ LocalRequired
-azmcp virtualdesktop hostpool sessionhost usersession-list --subscription <subscription> \
+azmcp virtualdesktop hostpool session user_list --subscription <subscription> \
```

Comment:
the underscore is inconsistent with L1915, L1916, L1917 for the same tool.  Change it back to a hyphen for consistency.